tasks/mssql_conf_setting.yml

```diff
@@ -0,0 +1,23 @@
+- name: Check if the {{ __mssql_conf_setting }} setting is enabled
+  command: >-
```

Comment:
you can't use `command` since you are piping, you'll have to use `shell`

however, I recommend using `command`, then parsing the output using ansible/jinja2 filters

you should also use the `quote` filter on the `command` arguments `__mssql_conf_setting` and `__mssql_conf_setting_value`

Reply:
Ok will do

Reply:
Changed this to not pipe and instead use ansible's regex_replace.
You are right about the quotes filter, I forgot to make this change with the last push. I've committed it locally and will push when the current tests pass.